ReactNativeClient/lib/services/rest/Api.js

```diff
@@ -451,6 +451,23 @@ class Api {
 			return note;
 		}
 
+		if (request.method === 'PUT') {
+			const note = await Note.load(id);
+			const requestBody = JSON.parse(request.body);
+
+			if (!note) throw new ErrorNotFound();
+
+			let newNote = Object.assign({}, note, request.bodyJson(this.readonlyProperties('PUT')));
```

Comment:
I'm not familiar with this interface so please help me understand. Is the `PUT` command intended to update an existing note?  Is this really a new note, or an updated one?

Reply:
Yes, the `PUT` command is intended to update an existing note. Perhaps, the variable name `newNote` was a bit confusing? I will change it to `updatedNote`.